src/vs/code/electron-main/backup.ts

```diff
@@ -0,0 +1,160 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+'use strict';
+
+import * as path from 'path';
+import * as crypto from 'crypto';
+import fs = require('fs');
+import Uri from 'vs/base/common/uri';
+import { readdirSync } from 'vs/base/node/extfs';
+import { IBackupWorkspacesFormat } from 'vs/platform/backup/common/backup';
+import { IEnvironmentService } from 'vs/platform/environment/common/environment';
+import { createDecorator } from 'vs/platform/instantiation/common/instantiation';
+
+export const IBackupService = createDecorator<IBackupService>('backupService');
+
+export interface IBackupService {
```

Comment:
I renamed it but kept it in electron-main like `IWindowsMainService` https://github.com/Microsoft/vscode/pull/14115/commits/78971e80cbd9413a2458d9325cffcee7359b43f5